svgs/general/code.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="none" d="M0 0h24v24H0V0z"/><path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"/></svg>
```

Comment:
remove `<path fill="none" d="M0 0h24v24H0V0z"/>` from this source and re-generate the icon component (or just remove that same line from the generated IconCode)

Reply:
@stefhatcher there it is...think the commit somehow lost when I tried rebasing

Reply:
aah! i see it now! thank you!